<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DesktopAppClass</class>
 <widget class="QWidget" name="DesktopAppClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1351</width>
    <height>911</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DesktopApp</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>1191</width>
     <height>881</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>View</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>741</width>
       <height>601</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGraphicsView" name="graphicsViewVideo"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QGraphicsView" name="graphicsViewVideo2"/>
        </item>
        <item>
         <widget class="QGraphicsView" name="graphicsViewVideo3"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Capture</string>
    </attribute>
    <widget class="QGraphicsView" name="graphicsViewVideo4">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>631</width>
       <height>321</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="captureButton">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>770</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>capture</string>
     </property>
    </widget>
    <widget class="QGraphicsView" name="graphicsViewImage">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>440</y>
       <width>591</width>
       <height>301</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="saveButtonCaptureTab">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>770</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>save</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>810</y>
       <width>323</width>
       <height>19</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="radioButton">
        <property name="text">
         <string>Color</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton2">
        <property name="text">
         <string>Depth</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton3">
        <property name="text">
         <string>Color to Depth</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton4">
        <property name="text">
         <string>Depth to Color</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="annotateButtonCaptureTab">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>770</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>AI analysis</string>
     </property>
    </widget>
    <widget class="QGraphicsView" name="graphicsViewVideo5">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>70</y>
       <width>381</width>
       <height>321</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="imuText">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>440</y>
       <width>411</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QGraphicsView" name="graphicsViewGyroscope">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>530</y>
       <width>191</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
    <widget class="QGraphicsView" name="graphicsViewAccelerometer">
     <property name="geometry">
      <rect>
       <x>910</x>
       <y>530</y>
       <width>191</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Annotate</string>
    </attribute>
    <widget class="QGraphicsView" name="graphicsViewAnnotation">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>401</width>
       <height>331</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="annotationsText">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>470</y>
       <width>271</width>
       <height>171</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>410</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="saveButtonAnnotateTab">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="annotateButtonAnnotateTab">
        <property name="text">
         <string>Fetch annotations</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="reportButton">
        <property name="text">
         <string>Generate report</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGraphicsView" name="graphicsViewAnnotation2">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>30</y>
       <width>401</width>
       <height>331</height>
      </rect>
     </property>
    </widget>
    <widget class="QGraphicsView" name="graphicsViewAnnotation_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>420</y>
       <width>401</width>
       <height>331</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="desktopapp.qrc"/>
 </resources>
 <connections/>
</ui>
